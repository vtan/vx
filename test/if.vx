(
  (let a 0)
  (let b 0)
  (let c 0)
  (let n 0)
  (let i 10)
  (while i (
    (if
      (= i 2) (
        (set a i)
        (set n (+ n 1)))
      (= i 5) (
        (set b i)
        (set n (+ n 1)))
      (= i 7) (
        (set c i)
        (set n (+ n 1)))
      (= i 1) (
        (set a (+ a 1))
        (set b (+ b 1))
        (set c (+ c 1))))
    (set i (- i 1))
  ))
  (set a (= a 3))
  (set b (= b 6))
  (set c (= c 8))
  (set n (= n 3))
)
