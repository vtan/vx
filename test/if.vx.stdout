00000000: 0000 0000 0000 0000 0100 0000 0000 0000  ................
00000010: 0100 0000 0000 0000 0100 0000 0000 0000  ................
00000020: 0100 0000 0000 0000                      ........
